src/docfx/build/metadata/SystemMetadata.cs

```diff
@@ -31,6 +31,8 @@ internal class SystemMetadata
         [JsonProperty("_op_gitContributorInformation")]
         public ContributionInfo? ContributionInfo { get; set; }
 
+        public string?[]? GitContributorInformation { get; set; }
```

Comment:
Consider name the metadata `github_contributors`:

- The existing `_op_gitContributorInformation` was named purely for backward compatibility, it does not adhere most of the naming conventions we had for metadata. Call this `git_contributor_information` would give you the false assumption that this is the same as `_op_gitContributorInformation` but it's actually not true, `author` is not included in the list.
- Remove the `_information` suffix because just about everything is a piece of information.
- The value is a list of _GitHub_ users not a list of _git_ users

Reply:
I will update the spec and create a new PR to fix it.